ladybug/windrose.py

```diff
@@ -0,0 +1,156 @@
+"""Ladybug windrose"""
+from ladybug.epw import EPW
+
+
+class WindRose(object):
+    """
+    Draws the Ladybug windrose
+    Usage:
+    """
+    __slots__ = ("epw", "HOYs", "annualHourlyData",
+                 "windCondition", "dataCondition",
+                 "numOfDirections", "centerPoint",
+                 "scale", "maxFrequency")
+
+    def __init__(self,
+                 epw_filePath,
+                 HOYs=range(0, 8760),
+                 annualHourlyData=None,
+                 windCondition=None,
+                 dataCondition=None,
+                 ):
+        """
+        Instantiates the Ladybug WindRose Object
+        """
+        self.epw = EPW(epw_filePath)
+        self.HOYs = HOYs
+        self.annualHourlyData = annualHourlyData
+        self.windCondition = self.check_condition(windCondition)
+        self.dataCondition = self.check_condition(dataCondition)
+
+    @staticmethod
+    def check_condition(condition):
```

Comment:
This can be removed as similar method is implemented for `DataCollection`. If you call DataCollection.filter_by_conditional_statement it will check_condition under the hood.